datastore/providers/weaviate_datastore.py

```diff
@@ -370,3 +369,17 @@ def _is_valid_weaviate_id(candidate_id: str) -> bool:
                 return True
         except ValueError:
             return False
+
+    @staticmethod
+    def _is_wcs_domain(url: str) -> bool:
```

Comment:
what if it is a WCS enterprise cluster? This regex won't work

Reply:
Good find! Let me get back to you on this

Reply:
done!